src/main/java/org/spongepowered/common/command/SpongeCommand.java

```diff
@@ -650,6 +665,173 @@ private static CommandSpec getTpsCommand() {
+                                    src.sendMessage(Text.of(TextColors.RED, "Failed to delete world!"));
+                                    SpongeImpl.getLogger().error("Failed to delete world: " + world.getWorldName());
+                                    return CommandResult.empty();
+                                }
+                            } catch (InterruptedException | ExecutionException e) {
+                                src.sendMessage(Text.of(TextColors.RED, "Failed to delete world! " + e.getMessage()));
+                                SpongeImpl.getLogger().error("Failed to delete world: " + world.getWorldName(), e);
+                                return CommandResult.empty();
+                            }
+
+                            return CommandResult.success();
+                        })
+                        .build(), "remove")
+                .child(CommandSpec.builder()
+                        .executor((src, args) -> {
+                            src.sendMessage(Text.of(TextColors.YELLOW, "*** Available Worlds ***"));
+                            src.sendMessage(Text.of(TextColors.GREEN, "(LOADED) ", TextColors.RED, "(NOT LOADED)"));
+
+                            final Text.Builder text = Text.builder();
+                            for (WorldProperties world : Sponge.getServer().getAllWorldProperties()) {
+                                if (Sponge.getServer().getWorld(world.getUniqueId()).isPresent()) {
+                                    text.append(Text.of(TextColors.GREEN, world.getWorldName()));
+                                } else {
+                                    text.append(Text.of(TextColors.RED, world.getWorldName()));
+                                }
+                                text.append(Text.of(", "));
+                            }
+
+                            src.sendMessage(text.build());
+                            return CommandResult.success();
+                        })
+                        .build(), "list")
+                .child(CommandSpec.builder()
+                        .arguments(
+                                onlyOne(string(Text.of("world")))
+                        )
+                        .executor((src, args) -> {
+                            final WorldProperties worldProperties = args.<WorldProperties>getOne(Text.of("world")).get();
+                            final Optional<World> world = Sponge.getServer().getWorld(worldProperties.getUniqueId());
+
+                            if (world.isPresent()) {
+                                final boolean raining = world.get().getWeather() == Weathers.RAIN;
+                                final boolean thundering = world.get().getWeather() == Weathers.THUNDER_STORM;
+                                src.sendMessage(Text.of("Chunks loaded: " + Iterables.size(world.get().getLoadedChunks())));
+                                // src.sendMessage(Text.of("Villages loaded: "));
```

Comment:
The commented out stuff were Canary things that couldn't be implemented on Sponge.